contracts/src/v0.8/liquiditymanager/test/LiquidityManager.setCrossChainRebalancer.t.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.24;
+
+import {IBridgeAdapter} from "../interfaces/IBridge.sol";
+import {ILiquidityManager} from "../interfaces/ILiquidityManager.sol";
+
+import {LiquidityManagerSetup} from "./LiquidityManagerSetup.t.sol";
+import {LiquidityManager} from "../LiquidityManager.sol";
+
+contract LiquidityManager_setCrossChainRebalancer is LiquidityManagerSetup {
+  event CrossChainRebalancerSet(
```

Comment:
can this event be imported from LiquidityManager directly instead of being redefined.

Reply:
This was not resolved